android/feature/find-meal/src/main/java/com/kinandcarta/feature/find/meal/viewmodel/DisplayMealsViewModel.kt

```diff
@@ -55,9 +55,11 @@ class DisplayMealsViewModel @ViewModelInject constructor(
     @RequiresPermission("android.permission.ACCESS_FINE_LOCATION")
     fun startUpdatingLocation() {
         fusedLocationProviderClient.lastLocation.addOnSuccessListener {
-            val latLng = LatLng(it.latitude, it.longitude)
-            updateUserLocation(latLng)
-            updateMeals(latLng)
+            if (it != null) {
```

Comment:
Is this null check necessary? Would we get to `startUpdatingLocation` if there was no current location?

Reply:
It was crashing on my side without checking if it's null.